src/stack/constants.py

```diff
@@ -19,6 +19,7 @@
 cluster_name_prefix = "bpi-"
 compose_deploy_type = "compose"
 compose_dir_name = "compose"
+compose_file_prefix = "composefile"
```

Comment:
What would you think to getting this from the ENV so we could override it for old stacks?

`BPI_COMPOSE_FILE_PREFIX=docker-compose`

Reply:
Good idea.